det3d/models/losses/centernet_loss.py

```diff
@@ -19,8 +19,9 @@ def forward(self, output, mask, ind, target):
     mask = mask.float().unsqueeze(2) 
 
     loss = F.l1_loss(pred*mask, target*mask, reduction='none')
-    loss = loss / (mask.sum() + 1e-4)
-    loss = loss.transpose(2 ,0).sum(dim=2).sum(dim=1)
+    loss = loss / (mask.sum(dim=(1, 2)).unsqueeze(1).unsqueeze(2) + 1e-4)
+    # loss: B * max_objects * dim(x,y,z,l,w,h,sin,cos)
+    loss = loss.transpose(2, 0).mean(dim=2).mean(dim=1)
```

Comment:
no need for second mean(dim=1) ? we can do sum there as we already divide by num_pos ?